### PR TITLE
Add obsevability addon

### PR DESCRIPTION
**Pull request by chivalryq**
**Observability addon usage**

- When not using PersistentVolumeClaim, just trun `alertmanager-pvc-enabled` and `server-pvc-enable` to false
```shell
vela addon enable observability alertmanager-pvc-enabled=false server-pvc-enabled=false grafana-domain=example.com
```
- When use PersistentVolumeClaim:
```shell
vela addon enable observability alertmanager-pvc-class=alicloud-disk-available alertmanager-pvc-size=50Gi server-pvc-class=alicloud-disk-available server-pvc-size=50Gi grafana-domain=example.com
```

note that:
1. `alertmanager-pvc-size`,`server-pvc-size` are optional, default to 20Gi. 

- [x] Add addon observability
- [x] Argument Usage

### CODE REVIEW

**Comment by zzxwill** [comment]
Should the command `vela addon enable observability alertmanager-pvc-enabled=false server-pvc-enabled=false ingress-domain=example.com` be changed to `vela addon enable observability -alertmanager-pvc-enabled=false -server-pvc-enabled=false -ingress-domain=example.com` with a stash?

**Comment by chivalryq** [comment]
> Should the command `vela addon enable observability alertmanager-pvc-enabled=false server-pvc-enabled=false ingress-domain=example.com` be changed to `vela addon enable observability -alertmanager-pvc-enabled=false -server-pvc-enabled=false -ingress-domain=example.com` with a stash?

Now it's designed to be "foo=bar" and no stash.

File: docs/examples/obervability/initializer-observability.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                domain: grafana.c276f4dac730c47b8b8988905e3c68fcf.cn-hongkong.alicontainer.com
+                http:
+                  "/": 80
+            - type: import-grafana-dashboard
+              properties:
+                grafanaServiceName: grafana
+                grafanaServiceNamespace: observability
+                credentialSecret: grafana
+                credentialSecretNamespace: observability
+                url: "https://raw.githubusercontent.com/zzxwill/grafana-dashboards/master/Kubernetes_for_Prometheus_Dashboard_CN_20201209_1628671317889.json"
```
Comment (zzxwill) [inline docs/examples/obervability/initializer-observability.yaml:74]: `Urls`.
https://github.com/oam-dev/kubevela/pull/1997/files#diff-5a75fa21b3796c21cee33d18081c90af754e2df8544e0add36b43ddeadc35d26R74-R76

File: docs/examples/obervability/trait-import-grafana-dashboard.cue [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		url: parameter.url
+	}
+}
+
+parameter: {
+	grafanaServiceName:        string
+	grafanaServiceNamespace:   *"default" | string
+	credentialSecret:          string
+	credentialSecretNamespace: *"default" | string
+	url:                       string
```
Comment (zzxwill) [inline docs/examples/obervability/trait-import-grafana-dashboard.cue:20]: https://github.com/oam-dev/kubevela/pull/1997/files#diff-6d3618aa4a4cd3de8f7def3822db5aeaa5137e8cdd45e37a4a902ff016e23ab3R11-R21

**Review by wonderflow — APPROVED** [review]

File: charts/vela-core/templates/addons/observability-asset.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+data:
+  initializer: |
+    apiVersion: core.oam.dev/v1beta1
+    kind: Initializer
+    metadata:
+      annotations:
+        addons.oam.dev/description: Preparations that observability need
+      name: observability-asset
+      namespace: vela-system
```
Comment (zzxwill) [inline charts/vela-core/templates/addons/observability-asset.yaml:10]:
```suggestion
      namespace: observability
```

File: charts/vela-core/templates/addons/observability.yaml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+      - ref:
+          apiVersion: core.oam.dev/v1beta1
+          kind: Initializer
+          name: fluxcd
+          namespace: vela-system
+      - ref:
+          apiVersion: core.oam.dev/v1beta1
+          kind: Initializer
+          name: observability-asset
+          namespace: vela-system
```
Comment (zzxwill) [inline charts/vela-core/templates/addons/observability.yaml:129]:
```suggestion
          namespace: observability
```

File: vela-templates/addons/auto-gen/observability-asset.yaml [hunk 5]
```diff
@@ -0,0 +1,116 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Initializer
+metadata:
+  annotations:
+    addons.oam.dev/description: Preparations that observability need
+  name: observability-asset
+  namespace: vela-system
```
Comment (zzxwill) [inline vela-templates/addons/auto-gen/observability-asset.yaml:7]:
```suggestion
  namespace: observability
```

File: vela-templates/addons/auto-gen/observability.yaml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  - ref:
+      apiVersion: core.oam.dev/v1beta1
+      kind: Initializer
+      name: fluxcd
+      namespace: vela-system
+  - ref:
+      apiVersion: core.oam.dev/v1beta1
+      kind: Initializer
+      name: observability-asset
+      namespace: vela-system
```
Comment (zzxwill) [inline vela-templates/addons/auto-gen/observability.yaml:126]:
```suggestion
      namespace: observability
```

File: vela-templates/addons/observability-asset/template.yaml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Initializer
+metadata:
+  annotations:
+    addons.oam.dev/description: "Preparations that observability need"
+  name: observability-asset
+  namespace: vela-system
```
Comment (zzxwill) [inline vela-templates/addons/observability-asset/template.yaml:7]:
```suggestion
  namespace: observability
```

File: vela-templates/addons/observability/template.yaml [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+    - ref:
+        apiVersion: core.oam.dev/v1beta1
+        kind: Initializer
+        name: fluxcd
+        namespace: vela-system
+    - ref:
+        apiVersion: core.oam.dev/v1beta1
+        kind: Initializer
+        name: observability-asset
+        namespace: vela-system
```
Comment (zzxwill) [inline vela-templates/addons/observability/template.yaml:19]:
```suggestion
        namespace: observability
```

**Review by zzxwill — APPROVED** [review]
lgtm

**Pull request merged**